src/url.c

```
@@ -357,6 +357,9 @@ bool get_oldest_url(uint8_t *key, uint8_t *subkey, int subkey_ln, uint8_t *data,
 		}
 		comp->identified = IDENTIFIED_NONE;
 		asset_declared(comp);
+		scanlog("get_oldest_url iter=%d purl[0]=%s identified=%d rank=%d release=%s\n",
+			iteration, comp->purls[0] ? comp->purls[0] : "(null)", comp->identified, comp->rank,
```

Cppcheck (2.21.0)

[warning] If memory allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfMemory)